Source/BaseTableDataDisplayManager/Support/BaseNonReusableTableCellGenerator.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NonReusableTableCellGenerator.swift
+//  ReactiveDataDisplayManager
+//
+//  Created by Alexander Filimonov on 02/03/2020.
+//  Copyright © 2020 Александр Кравченков. All rights reserved.
+//
+
+import Foundation
+
+/// Class for generating non-reusable Configurable UITableViewCell
+public class BaseNonReusableTableCellGenerator<Cell: Configurable>: TableCellGenerator, SelectableItem where Cell: UITableViewCell {
```

Comment:
Hello again) I like short class names, but this one it is the long one( 

@alexfilimon @LastSprint @chausovSurfStudio @dmitry2101 @fixique 

What do u think about renaming all Base-generators to something like:
BaseTableCellGenerator -> CellGenerator/ TCellGenerator 
BaseNonReusableTableCellGenerator -> NonReusableCellGenerator / NonReusableTCellGenerator
for tableview as we 95% cases we use tableviews not collections)

What do u think?)

Reply:
I am for naming generators directly for collections, because I use generators for `UIColectionView` as well as for `UITableView`. `CellGenerator` - for `UITableView` is not obvious. But `TCellGenerator` for `UITableView` maybe not so bad name, but what about `UICollectionView`? `CCellGenerator`?

Reply:
Well @chausovSurfStudio suggested that all "not obvious" names by default refers to UITableView)